.gitignore

```diff
@@ -0,0 +1 @@
+test/**/*_generated*
```

Comment:
NIT: Maybe add a new line at the end. Git has issues with non-terminated lines.

Reply:
New line added